linera-service/src/cli_wrappers/local_kubernetes_net.rs

```diff
@@ -40,8 +41,8 @@ pub struct LocalKubernetesNet {
     next_client_id: usize,
     tmp_dir: Arc<TempDir>,
     binaries_dir: Option<PathBuf>,
-    kubectl_instance: KubectlInstance,
-    kind_clusters: Vec<KindCluster>,
+    kubectl_instance: Arc<Mutex<KubectlInstance>>,
+    kind_clusters: Vec<Arc<Mutex<KindCluster>>>,
```

Comment:
Are the mutexes for the borrow checker or for synchronization purposes?

In the former case, one could just `clone` things.

Reply:
I see, will just clone `kind_clusters` then. Thank you for the pointer!